mycroft/skills/intent_service.py

```diff
@@ -17,13 +17,15 @@
 import time
 
 from mycroft.configuration import Configuration, set_default_lf_lang
+from mycroft.util.lang import set_active_lang
 from mycroft.util.log import LOG
 from mycroft.util.parse import normalize
 from mycroft.metrics import report_timing, Stopwatch
 from .intent_services import (
     AdaptService, AdaptIntent, FallbackService, PadatiousService, IntentMatch
 )
 from .intent_service_interface import open_intent_envelope
+from mycroft.messagebus.message import Message
```

Comment:
move up to where the other mycroft imports are

Reply:
done